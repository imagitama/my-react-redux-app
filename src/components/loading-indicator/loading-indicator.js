import React from 'react'
import styled from 'react-emotion'

const StyledLoadingIndicator = styled('div')({ 
  padding: '1rem',
  textAlign: 'center'
})

// TODO: Connect itself to state?
export const LoadingIndicator = () =>
  <StyledLoadingIndicator>
    Loading...
  </StyledLoadingIndicator>