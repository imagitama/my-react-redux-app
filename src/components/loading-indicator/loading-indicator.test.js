import React from 'react'
import { mount } from 'enzyme'
import { LoadingIndicator } from './loading-indicator'

it('Matches the snapshot', () => {
  expect(mount(<LoadingIndicator />)).toMatchSnapshot()
})