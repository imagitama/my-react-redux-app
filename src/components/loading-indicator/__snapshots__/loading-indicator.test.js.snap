// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<Component>
  <Styled(div)>
    <div
      className="css-os96a3"
    >
      Loading...
    </div>
  </Styled(div)>
</Component>
`;
